cardano-mempool/tests/Common.hs

```diff
@@ -0,0 +1,7 @@
+module Common
+  ( module X
+  ) where
+
+import Test.Hspec as X
```

Comment:
So far we've been using Tasty in cardano-base; if it's not prohibitive, I think it would be a good idea to rewrite this to use Tasty as well, for consistency and a smaller dependency footprint.

Reply:
Not a problem, will switch to tasty

Reply:
Converted to `tasty`, despite that `hspec` is way better :wink: